### ISSUE TITLE
The training loss

### ISSUE THREAD

**Issue by Vickeyhw** [megvii-research/mdistiller#53]
Thanks for your great work! When I run the code use:
`python3 tools/train.py --cfg configs/imagenet/r34_r18/dot.yaml`   
The training loss is much larger than the kd method in the first few epochs, and the test acc is also low, is it normal?
![1701193501070](https://github.com/megvii-research/mdistiller/assets/20540333/783ff253-1501-45d5-a0e9-1c6650e7f3b0)

**Comment by Zzzzz1** [comment]
The loss scale is too large. Did you change the batch-size or num-gpus?

**Comment by Vickeyhw** [comment]
@Zzzzz1 I use the original batch size 512 on 8 2080ti.   After re-ran the code, I got the following results:
![1701241395192](https://github.com/megvii-research/mdistiller/assets/20540333/13f996e4-0956-4bfb-9830-cedbb13e0bd5)
It seems still unstable and much worse than the vannila kd.

**Comment by JinYu1998** [comment]
@Vickeyhw  How long does it take you to run an epoch please, I find it very strange that it takes me 100 minutes to run a 1/4 Epoch on 8*3090.

**Comment by Vickeyhw** [comment]
@JinYu1998 23min/epoch.

**Comment by JinYu1998** [comment]
> @JinYu1998 23min/epoch.

Thanks for your response, I think I've identified the problem. Since my data is not on SSD, the io issue is causing slow training...
